src/lib/y2storage/setup_checker.rb

```diff
@@ -24,6 +24,8 @@
 require "y2storage/boot_requirements_checker"
 require "y2storage/proposal_settings"
 require "y2storage/with_security_policies"
+# memory size detection
+require "yast2/hw_detection"
```

Comment:
There is a method in yast-stoage-ng to get the RAM size: `StorageManager.instance.arch.ram_size`